examples/TestDriver/ios/TestDriver/AppDelegate.m

```diff
@@ -33,10 +33,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   [self.window makeKeyAndVisible];
 
   // Send events to local collector.
-  SEL setRootUrlSelector = @selector(setRootUrl:);
-  if ([[Heap class] respondsToSelector:setRootUrlSelector]) {
-    [[Heap class] performSelector:setRootUrlSelector withObject:@"http://localhost:3000"];
-  }
+//  SEL setRootUrlSelector = @selector(setRootUrl:);
```

Comment:
Maybe we should just do this with a top-level boolean so that we can just turn it on or off with a single line...